var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -12,30 +36,106 @@ class Adios(Package):
     """
 
     homepage = "http://www.olcf.ornl.gov/center-projects/adios/"
-    url = "https://github.com/ornladios/ADIOS/archive/v1.9.0.tar.gz"
+    url      = "https://github.com/ornladios/ADIOS/archive/v1.10.0.tar.gz"
+
+    # make -j is currently not supported
+    parallel = False
+
+    version('1.10.0', 'eff450a4c0130479417cfd63186957f3')
+    version('1.9.0' , '310ff02388bbaa2b1c1710ee970b5678')
+    version('1.8.0' , 'c9116ec31e6b386f0e6ea5ab675c57e9')
```

Comment:
Why do you need to add older versions than 1.10 at all?


On Mon, Aug 29, 2016 at 5:09 PM, Axel Huebl <notifications@github.com>
wrote:

> In var/spack/repos/builtin/packages/adios/package.py
> <https://github.com/LLNL/spack/pull/1652#discussion_r76687865>:
>
> > @@ -12,30 +36,106 @@ class Adios(Package):
> >      """
> >
> >      homepage = "http://www.olcf.ornl.gov/center-projects/adios/"
> > -    url = "https://github.com/ornladios/ADIOS/archive/v1.9.0.tar.gz"
> > +    url      = "https://github.com/ornladios/ADIOS/archive/v1.10.0.tar.gz"
> > +
> > +    # make -j is currently not supported
> > +    parallel = False
> > +
> > +    version('1.10.0', 'eff450a4c0130479417cfd63186957f3')
> > +    version('1.9.0' , '310ff02388bbaa2b1c1710ee970b5678')
> > +    version('1.8.0' , 'c9116ec31e6b386f0e6ea5ab675c57e9')
>
> I remember that some older versions on debian/ubuntu needed a -pthread
> patch in some of our systems (ornladios/ADIOS#68
> <https://github.com/ornladios/ADIOS/pull/68>).
>
> maybe I should only add 1.9.0 and 1.10.0 here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LLNL/spack/pull/1652/files/88254a14a7b45efcfbe12496cb6f68fe0d2156ae#r76687865>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADGMLckXIvllDxUOn-D6_SENt7MRod2Bks5qk0p-gaJpZM4Jvkjc>
> .
>